pkg/migration/volume.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migration
+
+import (
+	"github.com/pkg/errors"
+	apiErr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+// DeletePersistentVolumeClaim deletes PVC resource in which backup data will be kept and make sure it was removed
+func (m *Service) DeletePersistentVolumeClaim(name string) error {
+	klog.Info("Deleting PersistentVolumeClaim")
+	err := m.coreInterface.PersistentVolumeClaims(m.releaseNamespace).Delete(name, &metav1.DeleteOptions{})
+	if apiErr.IsNotFound(err) {
```

Comment:
I prefer to have this separately, first, it is more readable second I create a log which gives us some information in logs that PVC was removed before the method was called (in fact it is not possible because the job will not run without mounted PVC)